README.md

```diff
@@ -142,6 +142,9 @@ connection with your use of FFmpeg.
 - [Human Protocol](https://hmt.ai) uses CVAT as a way of adding annotation service to the human protocol.
 - [Cogito Tech LLC](https://bit.ly/3klT0h6), a Human-in-the-Loop Workforce Solutions Provider, used CVAT
   in annotation of about 5,000 images for a brand operating in the fashion segment.
+- [FiftyOne](https://fiftyone.ai) is an open-source dataset curation and model analysis tool for visualizing, exploring, and improving computer vision datasets and models that is [tightly integrated](https://voxel51.com/docs/fiftyone/integrations/cvat.html) with CVAT for annotation and label refinement.
+
+<a href="https://fiftyone.ai"><img src="https://user-images.githubusercontent.com/21222883/143083891-a75d9fba-2829-43b1-ac07-bd46860964b2.png" height="500" width="700" ></a>
```

Comment:
@ehofesmann , thanks for the contribution. Let's remove the screenshot. The sentence above looks great!